src/main/java/plates/Dimensions.java

```diff
@@ -0,0 +1,26 @@
+package plates;
+
+public class Dimensions {
+
+    int numberOfRows, numberOfColumns;
```

Comment:
Has to be `private`, and since these are set in the constructor and never changed - they can (and it's a good thing to do) be marked as `final`.